internal/cmd/request.go

```diff
@@ -0,0 +1,107 @@
+package cmd
+
+import (
+	"fmt"
+	"net"
+	"reflect"
+
+	"github.com/edulinq/autograder/internal/api/core"
+	"github.com/edulinq/autograder/internal/api/server"
+	"github.com/edulinq/autograder/internal/common"
+	"github.com/edulinq/autograder/internal/util"
+)
+
+type CommonOptions struct {
+	Verbose bool `help:"Use verbose output to show full request/response without specific formatting." default:"false"`
```

Comment:
"without specific formatting" doesn't really make sense.
You are outputting that data specifically with a specific format (JSON).
But what you want to say is that this may cause the output to not be a valid format/filetype.